app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragmentStore.kt

```diff
@@ -20,17 +20,22 @@ class BookmarkFragmentStore(
  * The complete state of the bookmarks tree and multi-selection mode
  * @property tree The current tree of bookmarks, if one is loaded
  * @property mode The current bookmark multi-selection mode
+ * @property guidBackstack A set of guids for bookmark nodes we have visited. Used to traverse back
+ *                  up the tree after a sync.
+ * @property isLoading true if bookmarks are still being loaded from disk
  */
 data class BookmarkFragmentState(
     val tree: BookmarkNode?,
     val mode: Mode = Mode.Normal(),
+    val guidBackstack: Set<String> = linkedSetOf(),
```

Comment:
It doesn't seem like we use any properties of a Set here. Maybe a list should be used instead?